flashcard_runner.rb

```diff
@@ -0,0 +1,31 @@
+require './lib/turn'
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+require 'rspec'
+
+
+card_1 = Card.new("What is 5 + 5?", "10", :STEM)
+card_2 = Card.new("What is Rachel's favorite animal?", "Rat", :TuringStaff)
+card_3 = Card.new("What is Mike's middle name?", "nobody knows", :TuringStaff)
+card_4 = Card.new("What cardboard cutout lives at Turing?", "Justin Bieber", :PopCulture)
+deck = Deck.new([card_1, card_2, card_3, card_4])
+round = Round.new(deck)   
+
+
+def start(round)
+  p "Welcome! You're playing with 4 cards."
```

Comment:
To make this a little more dynamic, you could interpolate the number of cards with something like this, in case there are ever more than 4 cards:
`"Welcome! You're playing with #{round.deck.count} cards."`